src/components/Tooltip/Tooltip.tsx

```diff
@@ -20,19 +20,18 @@ export interface TooltipProps extends Omit<ComponentProps<'div'>, 'content' | 's
 /**
  * @see https://floating-ui.com/docs/react-dom-interactions
  */
-export const Tooltip: FC<TooltipProps> = ({
-  animation = 'duration-300',
-  arrow = true,
-  children,
-  className,
-  content,
-  placement = 'top',
-  style = 'dark',
-  theme: customTheme = {},
-  trigger = 'hover',
-  ...props
-}) => {
-  const theme = mergeDeep(getTheme().tooltip, customTheme);
+export const Tooltip: Component<TooltipProps> = (p: TooltipProps) => {
+  const mergedProps = mergeProps({
+    animation: 'duration-300',
+    arrow: true,
+    placement: 'top',
+    style: 'dark',
+    theme: {},
+    trigger: 'hover'
+  }, p);
+  const [local, props] = splitProps(mergedProps, ['animation', 'arrow', 'children', 'className', 'content', 'placement', 'style', 'theme', 'trigger']);
```

Comment:
```suggestion
  const [local, props] = splitProps(mergedProps, ['style', 'trigger', 'children', 'theme']);
```

Not necessary to split props that aren't used by local.